### PR TITLE
jquery.dbEditorCombo tests

### PR DESCRIPTION
**Pull request by PeterChaplin**
image

### CODE REVIEW

**Comment by PeterChaplin** [comment]
Added test for "search" method:
image

File: test/jquery.dbEditorCombo.test.html [hunk 1]
```diff
@@ -0,0 +1,5 @@
+<div class="container" style="position:relative">
+  <div class="editor"
+       style="margin: 50px; border: 1px solid grey;"
+       >Fruit...</div>
```
Comment (bvw) [inline test/jquery.dbEditorCombo.test.html:4]: This is a non-standard html formatting technique.
Is there an agreed standard formatter with indentation , line continuation etc ?

File: test/jquery.dbEditorCombo.test.js [hunk 2]
```diff
@@ -0,0 +1,111 @@
+beforeEach(() => {
+    require('./mocks/offsetParent.js');
```
Comment (bvw) [inline test/jquery.dbEditorCombo.test.js:2]: Mocks have a specific meaning with Jest 
https://jestjs.io/docs/en/mock-functions

Can you justify using this term ?
Is your offsetParent function a polyfill ?

File: test/jquery.dbEditorCombo.test.html (hunk 1 above)
Comment (PeterChaplin) [reply test/jquery.dbEditorCombo.test.html:4]: Looking at https://developers.google.com/style/html-formatting and https://google.github.io/styleguide/htmlcssguide.html I'd say this is pretty standard, is there a specific change you'd prefer?

File: test/jquery.dbEditorCombo.test.js (hunk 2 above)
Comment (PeterChaplin) [reply test/jquery.dbEditorCombo.test.js]: Updated and re-tested with folder renamed to "polyfills"

**Pull request merged**